### DISCUSSION TITLE
Quick Share by Google

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#63] [General]
# Quick Share by Google

Check if Quick Share by Google is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/2bb988a2-0b86-4b82-8968-deefa00eafe4

**Comment by GTMoraes** [comment]
It is supported. The installer fails, but you can install it on a x86 machine, then copy the Quick Share installation folder to the ARM64 Windows, and it'll fully work, just fine.

**Reply by OpenSource03** [reply]
Updated.

**Reply by NoDebuff03** [reply]
Needs to be updated again, just used the installer it works perfect, also it is being emulated in windows (x64)

**Reply by ghost** [reply]
I know; I had submitted a "report an issue" form to say that the status is now emulated. In fact; this app has been covered by me and another user as well. I had written that this one should be removed and the other one that says native should be changed to emulated at the time of writing.

**Reply by OpenSource03** [reply]
Updated, thanks.
